src/components/TokenBottomSheet.tsx

```diff
@@ -122,20 +131,38 @@ function TokenBottomSheet<T extends TokenBalance>({
   title,
   titleStyle,
   TokenOptionComponent = TokenOption,
-}: Props<T>) {
+  showPriceUsdUnavailableWarning,
+}: TokenBottomSheetProps<T>) {
   const [searchTerm, setSearchTerm] = useState('')
+  const [selectedTokenNoUsdPrice, setSelectedTokenNoUsdPrice] = useState<T | null>(null)
 
   const { t } = useTranslation()
 
-  const onTokenPressed = (token: T) => () => {
+  const handleTokenPressed = (token: T) => () => {
```

Comment:
nit: maybe we should agree to use `onXXX` vs `handleXXX` and aim for consistency across components 😄